interfaces/Portalicious/src/app/pages/project/project-monitoring/project-monitoring.page.ts

```diff
@@ -96,7 +96,7 @@ export class ProjectMonitoringPageComponent {
       return;
     }
 
-    return this.payments.data()[0].payment;
+    return this.payments.data().length;
```

Comment:
Off the top of my head, this seems like a more reliable solution (I haven't tested it)

```suggestion
    return this.payments.data()[this.payments.data().length - 1].payment;
```

Reply:
I had this before, wasnt working
I've refactored the endpoint to return an ordered list now